backend/src/services/sumo_access/observation_access.py

```diff
@@ -0,0 +1,70 @@
+import logging
+from typing import List
+
+import json
+from fmu.sumo.explorer.objects.dictionary import Dictionary
+
+from ._helpers import SumoEnsemble
+from .observation_types import (
+    Observations,
+    SummaryVectorDateObservation,
+    SummaryVectorObservations,
+    ObservationType,
+    RftObservations,
+)
+
+LOGGER = logging.getLogger(__name__)
+
+
+class ObservationAccess(SumoEnsemble):
+    async def get_observations(self) -> Observations:
+        """Retrieve all observations found in sumo case"""
+        observations_collection = self._case.dictionaries.filter(
+            stage="case",
+            name="observations",
+            tagname="all",
+        )
+        if len(observations_collection) == 0:
+            return Observations()
+        if len(observations_collection) > 1:
+            raise ValueError(f"More than one observations dictionary found. {observations_collection.names}")
+
+        observations_handle: Dictionary = observations_collection[0]
+        observations_bytes = observations_handle.blob
+        observations_dict = json.loads(observations_bytes.decode())
```

Comment:
Some async method that should be used here as well? I do not know what they are called.